Neos.ContentRepository/Classes/Domain/NodeType/NodeTypeConstraints.php

```diff
@@ -78,14 +78,14 @@ public function getExplicitlyDisallowedNodeTypeNames(): array
 
     public function matches(NodeTypeName $nodeTypeName)
     {
-        // if $nodeTypeName is explicitely blacklisted, it is DENIED.
+        // if $nodeTypeName is explicitely excluded, it is DENIED.
         foreach ($this->explicitlyDisallowedNodeTypeNames as $disallowed) {
             if ((string)$nodeTypeName === (string)$disallowed) {
                 return false;
             }
         }
 
-        // if $nodeTypeName is explicitely whitelisted, it is ALLOWED.
+        // if $nodeTypeName is explicitely allowed, it is ALLOWED.
```

Comment:
That's what happens when you get rid of loaded terminology.

Reply:
Maybe we can make this a little bit clearer like:
```// if $nodeTypeName is explicitely allowed, it is ALLOWED. IT. IS. ALLLOOOOWED!!!1!1!!```

;)

Reply:
```suggestion
        // if $nodeTypeName is explicitely ALLOWED.
```